library/std/src/net/ip.rs

```diff
@@ -1214,13 +1214,33 @@ impl Ipv6Addr {
         u128::from_be_bytes(self.octets()) == u128::from_be_bytes(Ipv6Addr::LOCALHOST.octets())
     }
 
-    /// Returns [`true`] if the address appears to be globally routable.
+    /// Returns [`true`] if the address appears to be globally reachable
```

Comment:
"globally reachable" instead of "globally routable", as that is the terminology used in the Special Address Registry.